src/RAWcookedBundle/Resources/views/Default/index.html.twig

```diff
@@ -0,0 +1,290 @@
+{% extends 'RAWcookedBundle::base.html.twig' %}
+
+{% block title %}RAWcooked{% endblock %}
+
+{% block body %}
+<section>
+    <div class="row reorder-xs">
+        <div class="v-align carousel-padding-xs col-xs-12 col-sm-5">
+            <div>
+            {% image '@RAWcookedBundle/Resources/public/img/RAWcooked_256.png' %}
+                <img src="{{ asset_url }}" alt="RAWcooked logo" class="img-responsive center-xs" title="RAWcooked logo" />
+            {% endimage %}
+            </div>
+        </div>
+        <div class="v-align col-sm-7">
+            <div class="center-font-xs">
+                <h1 class="v-align bigtitle">RAWcooked</h1>
+            </div>
+            <div class="projectDescription center-font-xs">
+                <strong>RAWcooked</strong> encodes RAW audio-visual data into the Matroska container (MKV), using the video codec FFV1 for the image and audio codec FLAC for the sound. The metadata accompanying the RAW data are preserved, and sidecar files, like MD5, LUT or XML, can be added into the Matroska container as attachments. This allows to manage these audio-visual file format in an effective and transparent way (e.g. native playback by VLC), while saving typically between one and two thirds of the needed storage, and speeding up the back-up on LTO cartridges.<br/>
+                When needed, the uncompressed source is retrieved <b>bit-by-bit</b>, in a faster manner than with uncompressed source directly stored on LTO cartridges.
+            </div>
+            <br/>
+            <div class="projectDescription center-font-xs">
+                Sponsored by <a href="https://reto.ch">AV Preserve by reto.ch</a> with support from <a href="http://www.cna.public.lu">CNA (National Audiovisual Centre of Luxembourg)</a> and <a href="https://www.nb.no">Nasjonalbiblioteket (National Library of Norway)</a>; designed, led and fully developed by MediaArea.

```

Comment:
AV Preserv**ation** by reto.ch

Reply:
Will fix soon

Reply:
I should read all emails before answering :)

Reply:
AV Preserve is now AVP or… WeAreAVP ;-)